starter-code/main.js

```diff
@@ -53,6 +53,195 @@ mongoClient.connect(`mongodb://localhost:27017/crunchbase`, (error, db) => {
+              }
+            })
+            break;
+
+            case "5":
+              //5.- List by name all companies founded in the summer of 2004 (april to june) sorted by date.
+            db.collection('companies').find({ $and: [ {founded_year: 2004}, { founded_month: {$gt: 3}}, {founded_month: {$lt: 7}}]},{founded_month: 1}).count((error, result) => {
+              if (error) {
+                console.log(error);
+                rl.question(`\nType enter to continue: `, (answer) => { mainMenu() });
+              } else {
+                console.log(result);
+                rl.question(`\nType enter to continue: `, (answer) => { mainMenu() });
+              }
+            })
+            break;
+
+            case "6":
+              //6.- What companies have offices in "Barcelona".
+            db.collection('companies').find({ "offices.city": { $eq: 'Barcelona' } }).count((error, result) => {
+              if (error) {
+                console.log(error);
+                rl.question(`\nType enter to continue: `, (answer) => { mainMenu() });
+              } else {
+                console.log(result);
+                rl.question(`\nType enter to continue: `, (answer) => { mainMenu() });
+              }
+            })
+            break;
+
+            case "7":
+              //7.- List the 10 companies with more employees sorted ascending (show name and employees).
+            db.collection('companies').find({},{number_of_employees: 1}).limit(10).count((error, result) => {
+              if (error) {
+                console.log(error);
+                rl.question(`\nType enter to continue: `, (answer) => { mainMenu() });
+              } else {
+                console.log(result);
+                rl.question(`\nType enter to continue: `, (answer) => { mainMenu() });
+              }
+            })
+            break;
+
+            case "8":
+              //8.- Find the company with the name "Facebook"
+            db.collection('companies').find({name: "Facebook"}).count((error, result) => {
+              if (error) {
+                console.log(error);
+                rl.question(`\nType enter to continue: `, (answer) => { mainMenu() });
+              } else {
+                console.log(result);
+                rl.question(`\nType enter to continue: `, (answer) => { mainMenu() });
+              }
+            })
+            break;
+
+            case "9":
+              //9.- How many employees has Facebook?
+            db.collection('companies').find({name: 'Facebook'},{number_of_employees: 1}).count((error, result) => {
+              if (error) {
+                console.log(error);
+                rl.question(`\nType enter to continue: `, (answer) => { mainMenu() });
+              } else {
+                console.log(result);
+                rl.question(`\nType enter to continue: `, (answer) => { mainMenu() });
+              }
```

Comment:
Aquí la terminal te responde 1, porque en realidad solo has buscado una compañía, Facebook. 

Para responder a la pregunta de cuantos empleados tiene Facebook me hubiera valido este cambio.

```suggestion
              //9.- How many employees has Facebook?
            db.collection('companies').find({name: 'Facebook'},{number_of_employees: 1}).toArray((error, result) => {
              if (error) {
                console.log(error);
                rl.question(`\nType enter to continue: `, (answer) => { mainMenu() });
              } else {
                console.log(result[0].number_of_employees);
                rl.question(`\nType enter to continue: `, (answer) => { mainMenu() });
              }
```